tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-integration-test.php

```
@@ -212,16 +212,140 @@ public function test_register_hooks() {
+			'ids'
+		);
+		$this->assertSame(
+			count( $pending_before ),
+			count( $pending_after ),
+			'Repeated calls while the resync is pending must not stack queue entries.'
+		);
+
+		remove_action( 'wc_stripe_agentic_commerce_schedule_full_resync', [ $integration, 'schedule_full_resync_now' ] );
+		as_unschedule_all_actions( \WC_Stripe_Agentic_Commerce_Integration::SCHEDULED_ACTION, [], 'wc-stripe' );
+		as_unschedule_all_actions( \WC_Stripe_Agentic_Commerce_Integration::SCHEDULED_ACTION, [], 'wc-stripe-agentic-resync' );
+	}
```

PHPMD (2.15.0)

[error] The method test_schedule_full_resync_now_still_enqueues_when_recurring_is_pending is not named in camelCase. (undefined)

(CamelCaseMethodName)